File: FrontEnd/aseguradora/src/app/Models/Seguro.ts
import { Binary } from "@angular/compiler";

export class Seguro {
  SeguroId:number;
  NombreSeguro: string;
  Codigo: string;
  FechaCreacion: string;
  Suma: number;
  Prima: number;
  Estado: string;


  constructor() {
    this.SeguroId=0;
    this.NombreSeguro = '';
    this.Codigo = '';
    this.FechaCreacion= '';
    this.Suma = 0;
    this.Prima = 0;
    this.Estado='';
  }
}

File: BackEnd/Consultorio_Seguros_Chubb/Models/Seguro.cs
namespace Consultorio_Seguros_Chubb.Models
{
    public class Seguro
    {
            public int SeguroId { get; set; }
            public string nombre_seguro { get; set; }
            public string codigo { get; set; }
            public DateTime fecha_creacion { get; set; }
            public decimal suma { get; set; }
            public decimal prima { get; set; }
            public bool estado { get; set; }
       
    }
}
